pkg/server/handler/cluster/conversion.go

```diff
@@ -193,6 +193,11 @@ func (g *generator) defaultApplicationBundle(ctx context.Context) (*unikornv1.Ku
 		return nil, errors.OAuth2ServerError("unable to select an application bundle")
 	}
 
+	slices.SortFunc(applicationBundles.Items, func(a, b unikornv1.KubernetesClusterApplicationBundle) int {
```

Comment:
This is already done by the application bundle client, so why is this necessary?  Smells of code duplication.